xarray/core/datatree_mapping.py

```diff
@@ -62,12 +75,14 @@ def map_over_datasets(
     func : callable
         Function to apply to datasets with signature:
 
-        `func(*args: Dataset) -> Union[Dataset, tuple[Dataset, ...]]`.
+        `func(*args: Dataset, **kwargs) -> Union[Dataset, tuple[Dataset, ...]]`.
```

Comment:
Shouldn't it not be unpacked?

```suggestion
        `func(*args: Dataset, kwargs=kwargs) -> Union[Dataset, tuple[Dataset, ...]]`.
```

Reply:
I'm not really sure:

```python
def myfunc(ds, **kwargs):
    x = kwargs.pop("mult")
    return ds * mult

kwargs = {"mult:", 10.0}
newdt = dt.map_over_datasets(myfunc, kwargs=kwargs)
```
I'm thinking `kwargs=kwargs` on `map_over_datasets` but `**kwargs` on `func`. Does that make sense?

Reply:
Yes they are unpacked in `func` so I also think the current string is correct